app/controllers/util/FormAction.scala

```diff
@@ -25,13 +25,15 @@ sealed trait FormAction {
 object FormAction {
   private val addLabel = "Add"
   private val saveAndContinueLabel = "SaveAndContinue"
+  private val saveAndReturnLabel = "SaveAndReturn"
   private val continueLabel = "Continue"
   private val removeLabel = "Remove"
 
   private def fromUrlEncoded(input: Map[String, Seq[String]]): FormAction =
     input.flatMap {
       case (`addLabel`, _)             => Some(Add)
       case (`saveAndContinueLabel`, _) => Some(SaveAndContinue)
+      case (saveAndReturnLabel, _)     => Some(SaveAndReturn)
```

Comment:
Can you add ``` like in rest of the cases to be consistent with the actual code.

Reply:
Pretty sure this was already fixed wasnt it?